Fixes #34252 - Missing example for 'Install packages' field

stejskalleos opened the pull request:
<!---

Thank you for contributing to The Foreman, please read the
[following guide](https://www.theforeman.org/contribute.html), in short:

* [Create an issue](https://projects.theforeman.org/projects/foreman/issues)
* Reference the issue via `Fixes #1234` in the commit message
* Prefer present-tense, imperative-style commit messages
* Mark all strings for translation, see [1]
* Suggest prerequisites for testing and testing scenarios following example above.
* Prepend `[WIP]` for work in progress to prevent bots from triggering actions
* Be patient, we will do our best to take a look as soon as we can
* Explain the purpose of the PR, attach screenshots if applicable
* List all prerequisites for testing (e.g. VMware cluster, two smart proxies...)
* Reviewers often use extensive list of items to check, have a look prior submitting [2]
* Be nice and respectful

1: https://projects.theforeman.org/projects/foreman/wiki/Translating#Translating-for-developers
2: https://github.com/theforeman/foreman/blob/develop/developer_docs/pr_review.asciidoc
-->

stejskalleos committed:
Fixes #34252 - Missing example for 'Install packages' field

stejskalleos commented:
[test foreman]

ezr-ondrej reviewed (commented):
Why did we dropped the test? :thinking:
Not that it would be usefull much, but I do not see a reason to drop it in this PR (but I might be missing something).

stejskalleos commented:
> Why did we dropped the test? thinking

I did same in https://github.com/theforeman/foreman/pull/9026, in past we had discussion about the snapshot tests and how they provide little to none value so I didn't bother to update them and deleted them right away.
I don't have any strong opinion here, no problem to revert it back and update the snapshots.

ezr-ondrej reviewed (approved):
The snapshot is pretty useless, but we should provide some tests instead of these.
It still has some value - if the component would have some hard errors, the snapshot would fail.
So it gives you "it rendered". Which is not great and far from enough value for a file with 20 lines of text, but at least something :)

But that's just a general thought, I'm ok dropping this one, because it is very simple component :+1:

ofedoren commented on app/controllers/api/v2/registration_commands_controller.rb:
```diff
@@ -17,7 +17,7 @@ class RegistrationCommandsController < V2::BaseController
         param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours)")
         param :insecure, :bool, desc: N_("Enable insecure argument for the initial curl")
-        param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter. Example: `pkg1 pkg2`")
+        param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter, example: `pkg1 pkg2`.")
```
Well, that's my mistake I didn't notice, but we avoid the dots at the end of param descriptions. At least for the sake of consistency.

ezr-ondrej commented:
@stejskalleos can you address @ofedoren's concern in follow-up PR (with `Refs #34252..`) before we cherry-pick?

ezr-ondrej commented:
CP 3.1: 5e3d9f76b
